### PR TITLE
docs(components): [space]

### PR DESCRIPTION
**Pull request by chenxch**
* Adjust API style
* export SpaceInstance

image

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by ryuhangyeong — APPROVED** [review]

File: docs/en-US/component/space.md [hunk 1]
```diff
@@ -119,22 +119,24 @@ space/fill-ratio
...
+| class      | Classname                       | ^[string] / ^[object]                                                                                                           | -          |
+| direction  | Placement direction             | ^[string]`'vertical' \| 'horizontal'`                                                                                           | horizontal |
+| prefixCls  | Prefix for space-items          | ^[string]                                                                                                                       | -          |
+| style      | Extra style rules               | ^[string] / ^[object]`CSSProperties \| CSSProperties[] \| string[]`                                                             | -          |
+| spacer     | Spacer                          | ^[string] / ^[number] / ^[VNode]                                                                                                | -          |
+| size       | Spacing size                    | ^[string] / ^[number] / ^[object]`[horizontalSize?: number, verticalSize?: number]`                                             | small      |
```
Comment (tolking) [inline docs/en-US/component/space.md:134]:
```suggestion
| size       | Spacing size                    | ^[string]`'default' \| 'small' \| 'large'` / ^[number] / ^[enum]`[number, number]`                                             | small      |
```

File: docs/en-US/component/space.md [hunk 2]
```diff
@@ -119,22 +119,24 @@ space/fill-ratio
...
-
-## Space Slot
+## API
+
+### Attributes
+
+| Name       | Description                     | Type                                                                                                                            | Default    |
+| ---------- | ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | ---------- |
+| alignment  | Controls the alignment of items | ^[string]`'center' \| 'normal' \| 'stretch' \| ...` [align-items](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items) | center     |
+| class      | Classname                       | ^[string] / ^[object]                                                                                                           | -          |
```
Comment (tolking) [inline docs/en-US/component/space.md:129]:
```suggestion
| class      | Classname                       | ^[string] / ^[object] / ^[array]                                                                                                           | -          |
```

File: docs/en-US/component/space.md [hunk 3]
```diff
@@ -119,22 +119,24 @@ space/fill-ratio
...
+| direction  | Placement direction             | ^[string]`'vertical' \| 'horizontal'`                                                                                           | horizontal |
+| prefixCls  | Prefix for space-items          | ^[string]                                                                                                                       | -          |
+| style      | Extra style rules               | ^[string] / ^[object]`CSSProperties \| CSSProperties[] \| string[]`                                                             | -          |
```
Comment (tolking) [inline docs/en-US/component/space.md:132]: ^[object]`CSSProperties \| CSSProperties[] \| string[]`

Does this type seem reasonable?

Comment (chenxch) [reply docs/en-US/component/space.md:132]: At present, arrays and objects are unified into object types, and there is no good subdivision scheme.

Comment (chenxch) [reply docs/en-US/component/space.md:132]:
```
^[object] / ^[array]`CSSProperties[] \| string[]`
```
How about this?

**Review by tolking — APPROVED** [review]

**Pull request merged**